scripts/standardise_validate.py

```diff
@@ -47,10 +47,16 @@ def main() -> None:
     parser.add_argument("--cutline", dest="cutline", help="Optional cutline to cut imagery to", required=False, nargs="?")
     parser.add_argument("--collection-id", dest="collection_id", help="Unique id for collection", required=True)
     parser.add_argument(
-        "--start-datetime", dest="start_datetime", help="Start datetime in format YYYY-MM-DD", type=valid_date, required=True
+        "--start-datetime",
+        dest="start_datetime",
+        help="Start datetime in format YYYY-MM-DD. Only optional if includeDerived.",
+        type=valid_date,
```

Comment:
Why not make this `required=False` instead of allowing `--start-datetime=`?

Reply:
Is the default value not `False`?

Reply:
Sorry, I did forgot to revert that before PR